setup.cfg

```diff
@@ -64,7 +64,7 @@ tests =
 web =
     flask
     flask-wtf
-    bootstrap-flask
+    bootstrap-flask<=2.0.0
```

Comment:
We might want to add a lower bound as well depending on when Bootstrap4 was added - this didn't work with my 1.7.0 version.

Reply:
Or did you actually mean to put `>=` here?

Reply:
I concluded that the only logical explanation is that it should be >=

Reply:
Thanks for the typo spot, you were correct